package.json

```diff
@@ -7,7 +7,7 @@
   "license": "MIT",
   "scripts": {
     "postinstall": "bin/link-self",
-    "start": "node demo/stripes.js dev demo/stripes.config.js",
+    "start": "NODE_ENV=development node demo/stripes.js dev demo/stripes.config.js",
```

Comment:
Without this change:
<img width="531" alt="screen shot 2017-12-05 at 5 25 46 pm" src="https://user-images.githubusercontent.com/230597/33640141-a417c5a0-d9e3-11e7-9bb2-4f7f47495879.png">